src/domain-registry-essentials/templates/cns-smart-contracts-renderer.ts

```diff
@@ -0,0 +1,98 @@
+import Fs from 'fs';
+import Ejs from 'ejs';
+import MarkdownInclude from 'markdown-include';
+import NetworkConfigJson from 'dot-crypto/src/network-config/network-config.json';
+
+const TemplatesDir = 'src/domain-registry-essentials/templates';
+const ContractsDir = TemplatesDir + '/contracts';
+const MarkdownFile = TemplatesDir + '/markdown/cns-smart-contracts-markdown.json';
+const TemplateToRender = TemplatesDir + '/cns-smart-contracts-template.md';
+const FileToRender = 'src/domain-registry-essentials/cns-smart-contracts.md';
+const CotractTableTemplate = Fs.readFileSync(ContractsDir + '/contract-table-template.ejs', 'utf-8');
+const Networks: Record<number, string> = {
+    1: 'Mainnet',
+    3: 'Ropsten',
+    4: 'Rinkeby',
+    5: 'Goerli',
+    42: 'Kovan',
+};
+
+type Row = {
+    network: string,
+    address: string,
+    legacyAddresses: [number],
+}
+
+renderContractAddresses();
+
+function renderContractAddresses() {
+    console.log('Rendering [' + FileToRender + ']');
+    console.log('From template [' + TemplateToRender + ']');
+
+    let filesToInclude: string[] = [TemplateToRender];
+
+    for (let contract of getContracts()) {
+        generateContractTables(contract as string, filesToInclude);
```

Comment:
If you specify the return type for `getContracts()`, you won't need to cast `contract` to `string`.

Reply:
✅